<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1432</width>
    <height>923</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>walberla GUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/resources/MolecularDynamics.png</normaloff>:/icons/resources/MolecularDynamics.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1432</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOtherConfigs">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Other Configs</string>
     </property>
     <addaction name="actionExample_Entry"/>
    </widget>
    <addaction name="actionOpenTestcase"/>
    <addaction name="actionSaveConfig"/>
    <addaction name="actionSaveConfigAs"/>
    <addaction name="menuOtherConfigs"/>
    <addaction name="separator"/>
    <addaction name="actionSaveVTKFile"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>Control</string>
    </property>
    <addaction name="actionForward"/>
    <addaction name="actionFastForward"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionRun"/>
    <addaction name="actionPause"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuViews">
    <property name="title">
     <string>Views</string>
    </property>
    <addaction name="actionNewGridView"/>
    <addaction name="separator"/>
    <addaction name="actionTile"/>
    <addaction name="actionCascade"/>
    <addaction name="actionFullscreen"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuControl"/>
   <addaction name="menuViews"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="controlBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRun"/>
   <addaction name="actionPause"/>
   <addaction name="actionForward"/>
   <addaction name="actionFastForward"/>
   <addaction name="separator"/>
   <addaction name="actionNewGridView"/>
   <addaction name="actionNewTreeView"/>
   <addaction name="actionContinue"/>
  </widget>
  <widget class="QDockWidget" name="dockBlocks3D">
   <property name="windowTitle">
    <string>Blocks (3D View)</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_4"/>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockControl">
   <property name="windowTitle">
    <string>Control</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="4" column="1">
      <widget class="QLabel" name="lblTimestepsExecuted">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spnTimeStepsForward">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999999</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Timesteps executed:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="inpTimerInterval">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Timer Interval</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Forward Time-Steps:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spnTimeStepsFastForward">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>9999999</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Fast Forward Time-Steps:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QCheckBox" name="chkUpdateView">
       <property name="text">
        <string>Update View after each timestep</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockSliceProperties">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Grid View Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QStackedWidget" name="propertyStack">
       <widget class="QWidget" name="page"/>
       <widget class="QWidget" name="page_2"/>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockBreakpoints">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Breakpoints</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="chkOnlyTimestep">
       <property name="text">
        <string>Only in Timestep</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spnOnlyTimestep">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="chkOnlyLocation">
       <property name="text">
        <string>File has to contain:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtOnlyLocation">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Location:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="lblLocation">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Messsage</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="lblBreakpointMsg">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QToolButton" name="cmdContinue">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Continue</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icons/resources/start-here.png</normaloff>:/icons/resources/start-here.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>243</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockBlocksText">
   <property name="windowTitle">
    <string>Blocks (Text)</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5"/>
  </widget>
  <action name="actionSaveParticleFile">
   <property name="icon">
    <iconset>
     <normaloff>:/qwt3d_images/scattered.png</normaloff>:/qwt3d_images/scattered.png</iconset>
   </property>
   <property name="text">
    <string>Save Particle File...</string>
   </property>
  </action>
  <action name="actionSaveVTKFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/paraview.png</normaloff>:/icons/resources/paraview.png</iconset>
   </property>
   <property name="text">
    <string>Save VTK File...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/application-exit.png</normaloff>:/icons/resources/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpenTestcase">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/document-open.png</normaloff>:/icons/resources/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open Testcase...</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/1rightarrow.png</normaloff>:/icons/resources/1rightarrow.png</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
  <action name="actionFastForward">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/2rightarrow.png</normaloff>:/icons/resources/2rightarrow.png</iconset>
   </property>
   <property name="text">
    <string>Fast Forward</string>
   </property>
   <property name="toolTip">
    <string>Do Multiple TimeSteps</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/media-playback-start.png</normaloff>:/icons/resources/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/media-playback-pause.png</normaloff>:/icons/resources/media-playback-pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="actionNewGridView">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/media_playlist.png</normaloff>:/icons/resources/media_playlist.png</iconset>
   </property>
   <property name="text">
    <string>New Grid View</string>
   </property>
  </action>
  <action name="actionSaveConfigAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/document-save-as.png</normaloff>:/icons/resources/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save Config As</string>
   </property>
  </action>
  <action name="actionSaveConfig">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/document-save.png</normaloff>:/icons/resources/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save Config</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/media_playlist.png</normaloff>:/icons/resources/media_playlist.png</iconset>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionExample_Entry">
   <property name="text">
    <string>Example Entry</string>
   </property>
  </action>
  <action name="actionNewTreeView">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/messagebox_info.png</normaloff>:/icons/resources/messagebox_info.png</iconset>
   </property>
   <property name="text">
    <string>New Tree View</string>
   </property>
  </action>
  <action name="actionContinue">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/start-here.png</normaloff>:/icons/resources/start-here.png</iconset>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkOnlyLocation</sender>
   <signal>toggled(bool)</signal>
   <receiver>txtOnlyLocation</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1078</x>
     <y>583</y>
    </hint>
    <hint type="destinationlabel">
     <x>1291</x>
     <y>583</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkOnlyTimestep</sender>
   <signal>toggled(bool)</signal>
   <receiver>spnOnlyTimestep</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1078</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>1291</x>
     <y>556</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
